<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switchAccountDlg</class>
 <widget class="QDialog" name="switchAccountDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>switchAccountDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{
background:#FFFFFF;
border-radius:2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#lbl_name{
font-family:&quot;微软雅黑&quot;;
font-size:24px;
color:#142B59;
}
#btn_login{
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#FFFFFF;
background:#142B59;
}
#btn_login:pressed{
margin:1px;
}
#wgt_account,#wgt_passwd{
border-radius:2px;
border:1px solid rgba(222,238,242,1);
background:rgba(255,255,255,1);
box-shadow:0px 0px 4px 0px rgba(237,244,250,0.54); 
}
#lbl_account{
background-image:url(:/masterCashier/Resources/login/login_account_black.png);
}
#lbl_passwd{
background-image:url(:/masterCashier/Resources/login/login_passwd_black.png);
}
#lEdit_account,#lEdit_passwd{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(0,0,0,0.7);
background:transparent;
border:none;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lbl_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>122</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QWidget" name="wgt_account" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_account">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEdit_account">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>我的账号</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wgt_passwd" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_passwd">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEdit_passwd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>请输入密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_login">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>122</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_hint">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#DE0000;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
